src/test/java/teammates/test/pageobjects/GoogleLoginPage.java

```diff
@@ -12,7 +12,8 @@
 
 public class GoogleLoginPage extends LoginPage {
 
-    private static final String EXPECTED_SNIPPET_SIGN_IN = "Sign in - Google Accounts";
+    private static final String EXPECTED_SNIPPET_SIGN_IN_VARIANT_ONE = "Sign in - Google Accounts";
+    private static final String EXPECTED_SNIPPET_SIGN_IN_VARIANT_TWO = "Sign in - Google accounts";
```

Comment:
Since this is the current variant Google uses, I suggest naming this line as `EXPECTED_SNIPPET_SIGN_IN_CURRENT_VARIANT` and the one on line 15 as `EXPECTED_SNIPPET_SIGN_IN_BACKUP_VARIANT`. Your thoughts on this? @xpdavid

Reply:
@carsonshoupe I just want to point out this. Actually, the original issue is updated 18 days ago. 

> It seems that Google has changed the title and the behaviour become consistent.

The new title should be `Sign in – Google accounts`.

Reply:
Ah.  That makes sense.  I'll make the updates and push the branch.